### ISSUE TITLE
Positron panels are drawn over their focus outlines, overlapping and/or clipping them

### ISSUE THREAD

**Issue by jmcphers** [posit-dev/positron#8929]
#### Positron and OS details:

```
Positron Version: 2025.09.0 build 26
Code - OSS Version: 1.102.0
Commit: 44a1d19be2e1cd01c5c6ae6cb37b6b3117f9c203
Date: 2025-08-07T03:42:32.533Z
Electron: 35.6.0
Chromium: 134.0.6998.205
Node.js: 22.15.1
V8: 13.4.114.21-electron.0
OS: Darwin arm64 24.6.0
```

#### Session details:

N/A

## Describe the issue:

Many Positron panels are drawn on top of their focus outlines. When focused, only a partial outline appears. This looks sloppy, since part of the focus outline draws, but not the whole thing. It may also be an accessibility issue. Examples:

### Plots pane

Missing most of the left and right focus outline; overlapped by the image.

<img width="1314" height="998" alt="Image" src="https://github.com/user-attachments/assets/19c3126e-6abf-4006-99fd-c26bd6145915" />

### Console tab list

Missing the top:

<img width="212" height="340" alt="Image" src="https://github.com/user-attachments/assets/3eea115e-cd8f-49e9-a872-a63e2c643606" />

### Variables

Missing most of the left, right, and top; kind of a weird long U thing is drawn at the bottom

<img width="661" height="227" alt="Image" src="https://github.com/user-attachments/assets/74c146fc-3843-4043-952b-4dd35188f83d" />

## Expected or desired behavior:

A focus outline should be a complete rectangle. 

For example, VS Code's _Problems_ pane unfocused:

<img width="684" height="351" alt="Image" src="https://github.com/user-attachments/assets/fd370e7a-5a66-4682-8392-bff3262ab60f" />

And focused:

<img width="679" height="347" alt="Image" src="https://github.com/user-attachments/assets/2e6cba48-dc35-4bc5-be0e-13fab22c9f6f" />

**Comment by juliasilge** [comment]
Related to #3171
